docs/graphql/manual/api-reference/schema-metadata-api/event-triggers.rst

```diff
@@ -224,3 +224,77 @@ EventTriggerColumns
    :class: haskell-pre
 
    "*" | [:ref:`PGColumn`]
+
+.. _pause_event_trigger:
+
+pause_event_trigger
+--------------------
```

Comment:
```suggestion
-------------------
```

Reply:
Underline should always be the same length as the title 🙂

Reply:
Oh got it, thank you 😄